gravitee-apim-rest-api/gravitee-apim-rest-api-security/src/main/java/io/gravitee/rest/api/security/csrf/CookieCsrfSignedTokenRepository.java

```diff
@@ -79,7 +78,10 @@ public CsrfToken generateToken(HttpServletRequest request) {
     }
 
     @Override
+    @SneakyThrows
```

Comment:
Can't you re throw something ?

Reply:
I've moved the initialization of the signer and verifier which can throw an exception if missconfigured. I choose to simply rethrow the exception coming from nimbus. Do you see any issue?